src/Components/utils.ts

```diff
@@ -0,0 +1,35 @@
+export const saveToLocal = () => {
```

Comment:
Is this fn used? (if not, add a comment above saying so if you want to keep the code)